### PR TITLE
Add localization comments to settings UI

### PR DESCRIPTION
**Pull request by carlos-zamora**
## Summary of the Pull Request
We've received feedback from the localization team that the strings need some comments to provide more context.

## References
Closes [MSFT-31929817](https://microsoft.visualstudio.com/OS/_workitems/edit/31929817)
#9328 - Chinese translation issues
#9238 - French translation issues

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -119,281 +119,363 @@
...
+    <comment>Header for a control to toggle if the app should always show the tabs (similar to a website browser).</comment>
   </data>
   <data name="Globals_AlwaysShowTabs.HelpText" xml:space="preserve">
     <value>When unchecked, the tab bar will appear when a new tab is created.</value>
-  </data>
-  <data name="Globals_CopyFormatting.Content" xml:space="preserve">
-    <value>Copy formatting</value>
-  </data>
-  <data name="Globals_CopyFormatting.HelpText" xml:space="preserve">
-    <value>When checked, the color and font formatting of selected text is also copied to your clipboard. When unchecked, only plain text is copied to your clipboard.</value>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:190]: unused?

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -119,281 +119,363 @@
...
   <data name="Profile_AltGrAliasing.Header" xml:space="preserve">
     <value>AltGr aliasing</value>
+    <comment>Header for a control to toggle whether the app treats ctrl+alt as the AltGr (also known as the Alt Graph) modifier key found on keyboards. </comment>
   </data>
   <data name="Profile_AltGrAliasing.HelpText" xml:space="preserve">
     <value>By default Windows treats Ctrl+Alt as an alias for AltGr. When unchecked, this behavior will be disabled.</value>
+    <comment>A description for what the "AltGr aliasing" setting does. Presented near "Profile_AltGrAliasing.Header".</comment>
   </data>
   <data name="Profile_AntialiasingMode.Header" xml:space="preserve">
     <value>Text antialiasing (requires relaunch)</value>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:441]: This isn't relevant to this PR, but this setting is hot-reloadable...

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 3]
```diff
@@ -433,236 +515,303 @@
...
+    <comment>An option to choose from for the "launch mode" setting. Opens the app maximized and in focus mode.</comment>
   </data>
   <data name="Profile_BellStyle.Header" xml:space="preserve">
     <value>Bell notification style</value>
+    <comment>Header for a control to select the how the app notifies the user. "Bell" is the common term in terminals for the BEL character (like the metal device used to chime).</comment>
   </data>
   <data name="Profile_BellStyle.HelpText" xml:space="preserve">
     <value>Controls what happens when the application emits a BEL character.</value>
-    <comment>"Audible" and "None" must match the values for Profile_BellStyleAudible.Content and Profile_BellStyleNone.Content respectively.</comment>
+    <comment>A description for what the "bell style" setting does. Presented near "Profile_BellStyle".</comment>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:802]: Should `BEL` be locked? Presumably that's universal...

**Review by zadjii-msft — APPROVED** [review]
I mean, I'm not sure exactly what they're looking for, but this looks good to me.

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw (hunk 1 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:190]: Yup! I think at one point we thought this was going to be a toggle. But instead we just created the radio button approach.

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw (hunk 2 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:441]: Removing the requires relaunch thing then.

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 4]
```diff
@@ -119,281 +119,363 @@
   </resheader>
   <data name="ColorScheme_Background.Text" xml:space="preserve">
     <value>Background</value>
+    <comment>The background color for text displayed on the screen. Users can select a color to use in this context.</comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:122]: Sorry, these need to be more about the _string_, not about what the setting does. "This is the header for a control that lets the user select the background color for text displayed on the screen."

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 5]
```diff
@@ -119,281 +119,363 @@
...
     <value>When unchecked, the title bar will appear above the tabs.</value>
+    <comment>A description for what the "show titlebar" setting does. Presented near "Globals_ShowTitlebar.Header".</comment>
   </data>
   <data name="Globals_ShowTitleInTitlebar.Header" xml:space="preserve">
     <value>Use active terminal title as application title</value>
+    <comment>Header for a control to toggle whether the terminal's title is shown as the application title, or not.</comment>
   </data>
   <data name="Globals_ShowTitleInTitlebar.HelpText" xml:space="preserve">
     <value>When unchecked, the title bar will be 'Windows Terminal'.</value>
+    <comment>A description for what the "show title in titlebar" setting does. Presented near "Globals_ShowTitleInTitlebar.Header".</comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:306]: lock Windows Terminal

Comment (DHowett) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:306]: or lock Windows i guess?

Comment (DHowett) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:306]: I don't think we localize the title ...

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 6]
```diff
@@ -119,281 +119,363 @@
...
     <value>Sets the transparency of the window.</value>
+    <comment>A description for what the "acrylic opacity" setting does. Presented near "Profile_AcrylicOpacity.Header".</comment>
   </data>
   <data name="Profile_Advanced.Header" xml:space="preserve">
     <value>Advanced</value>
+    <comment>Header for a sub-page of profile settings focused on more advanced scenarios.</comment>
   </data>
   <data name="Profile_AltGrAliasing.Header" xml:space="preserve">
     <value>AltGr aliasing</value>
+    <comment>Header for a control to toggle whether the app treats ctrl+alt as the AltGr (also known as the Alt Graph) modifier key found on keyboards. </comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:434]: lock AltGr maybe??

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 7]
```diff
@@ -119,281 +119,363 @@
...
     <value>Controls how text is antialiased in the renderer.</value>
+    <comment>A description for what the "antialiasing mode" setting does. Presented near "Profile_AntialiasingMode.Header".</comment>
   </data>
   <data name="Profile_AntialiasingModeAliased.Content" xml:space="preserve">
     <value>Aliased</value>
+    <comment>An option to choose from for the "text antialiasing" setting. When selected, the app's text renderer does not use antialiasing techniques.</comment>
   </data>
   <data name="Profile_AntialiasingModeClearType.Content" xml:space="preserve">
     <value>ClearType</value>
+    <comment>An option to choose from for the "text antialiasing" setting. When selected, the app's text renderer uses a "ClearType" antialiasing technique. "ClearType" is a proper noun.</comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:454]: do we lock ClearType? I don't know what Windows does.

**Review by DHowett — CHANGES_REQUESTED** [review]
this is great! just want to reword some of the color ones!

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw (hunk 7 above)
Comment (DHowett) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:454]: The "Cleartype text tuner" _locks the term ClearType_!!!

**Comment by skyline75489** [comment]
My honest opinion of the current translation is they are mainly out of context, for example using a chemical word to translate “acrylic”. 

Also I think some of the translations requires certain knowledge about Terminal. I hope my recommendation in #9328 helps. I’m really looking forward to a well-translated interface in 1.7.

**Comment by DHowett** [comment]
@skyline75489 yes, when i reported the french localization issue they discovered that there were no comments and filed a bug on us to add a bunch of comments to provide context. This will help us get a very well localized interface :)

**Comment by DHowett** [comment]
I will send them your suggestions as well. Thank you for looking up the resource keys -- it makes my job MUCH easier!

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 8]
```diff
@@ -119,7 +119,7 @@
   </resheader>
   <data name="ColorScheme_Background.Text" xml:space="preserve">
     <value>Background</value>
-    <comment>The background color for text displayed on the screen. Users can select a color to use in this context.</comment>
+    <comment>This is the header for a control that lets the user select the background color for text displayed on the screen.</comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:122]: i was really hoping you would apply this to all of the color comments that were phrased the same way... >_<

Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:122]: derp hahaha. Onnit

**Review by DHowett — APPROVED** [review]
thank you!

**Comment by moonlightz** [comment]
I have portuguese from portugal set as language ui. However, Windows Terminal, Edge's Dev Tools and some other parts are portuguese brazillian translated. I wonder when Windows Terminal will get portuguese portugal translation...

**Pull request merged**